xsd/netex_framework/netex_reusableComponents/netex_vehicle_version.xsd

```diff
@@ -185,6 +185,11 @@ Rail transport, Roads and Road transport
 					<xsd:documentation>ACTUAL EQUIPMENT found in VEHICLE.</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
+			<xsd:element name="Monitored" type="xsd:boolean" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Whether real-time data is available for the VEHICLE (this can be the vehicle position, but also the speed, charging level, passenger counting, etc.). Overrides the corresponding value in VehicleType. </xsd:documentation>
```

Comment:
```suggestion
					<xsd:documentation>Whether real-time data is available for the VEHICLE (this can be the vehicle position, but also the speed, charging level, passenger counting, etc.). Overrides the corresponding value in VehicleType.</xsd:documentation>
```